app/hotels/src/singleHotel/PaymentScreen.js

```diff
@@ -57,12 +57,18 @@ export class PaymentScreen extends React.Component<PaymentParameters> {
     };
   };
   componentDidMount = () => {
-    Logger.ancillaryDisplayed(Logger.Type.ANCILLARY_STEP_PAYMENT);
+    Logger.ancillaryDisplayed(
+      Logger.Type.ANCILLARY_STEP_PAYMENT,
+      Logger.Provider.ANCILLARY_PROVIDER_BOOKINGCOM,
```

Comment:
This could be wrong. I think you need to wait for my PR, then you can fetch the apiProvider from the context

Reply:
For now, it seems like the PaymentScreen is only for Booking.com (the graphql query is 
```
graphql`
          query PaymentScreenQuery(
            $hotelId: ID
            $roomConfig: [RoomConfigInput]
          ) {
            hotelPaymentUrls(hotelId: $hotelId, roomConfig: $roomConfig) {
              bookingComPaymentUrl
            }
          }
        `
```

Reply:
Yeah, you are right, I will improve this when I add support for stay22.